CHANGES.rst

```diff
@@ -2,6 +2,25 @@
 Changes
 =======
 
+**2018-??-??**
+*Released 11.0.0*
+
+Context based callbacks:
+See https://git.io/vp113 for help.
```

Comment:
Not correct url anymore
applies elsewhere too

Reply:
Fixed it in #1221